services/QuillLMS/app/models/user.rb

```diff
@@ -234,7 +234,8 @@ class User < ApplicationRecord
 
   validates :password,
     presence: { if: :requires_password? },
-    length: { maximum: CHAR_FIELD_MAX_LENGTH }
+    length: { maximum: CHAR_FIELD_MAX_LENGTH },
+    unless: -> { validation_context == :classlink_user }
```

Comment:
I'm not sure this thought is an improvement, but should we consider making `:classlink_user` a constant of some sort with the same thought as usual: it helps avoid typo-ing the value when used in a call?

Reply:
Haven't seen `validation_context` before  😎

Reply:
I think it be instructive to add a test in user_spec.rb that shows this `validation_context` in action.